.github/workflows/smithy-polymorph.yml

```diff
@@ -31,11 +31,17 @@ jobs:
           # Matching the hard-coded version for the "2023" edition for now
           dafny-version: 4.1.0
 
-      - name: Install smithy-dafny-codegen dependencies locally
+      - name: Install smithy-dafny-codegen Rust dependencies locally
         uses: gradle/gradle-build-action@v2
         with:
           arguments: :codegen-client:pTML :codegen-core:pTML :rust-runtime:pTML
           build-root-directory: smithy-dafny-codegen-modules/smithy-rs
+      
+      - name: Install smithy-dafny-codegen Python dependencies locally
```

Comment:
Just FYI I had to do this because `smithy-rs` packages aren't published at all yet, but I assume you have to do it here because you're still patching the `smithy-python` code?

Reply:
Also it might be worth a reusable `.github/actions` for this since it's not very DRY.

Reply:
Yes and yes (both because Smithy-Python isn't on Maven *and* because I have patches.)

Reply:
I'm looking at this and don't see a *ton* of value in abstracting this to a resuable GHA?
I might hold off on this for now unless you think it's blocking.

Reply:
Made a GHA for "Install Smithy-Dafny codegen dependencies"

Reply:
FYI I snuck these targets in for human usage too: https://github.com/smithy-lang/smithy-dafny/blob/main-1.x/Makefile#L13 :)